cypress/e2e/immutableDatabaseTests/workflowDetails.ts

```diff
@@ -126,3 +127,16 @@ describe('Find workflow by alias', () => {
     cy.url().should('eq', Cypress.config().baseUrl + '/workflows/github.com/A/l');
   });
 });
+
+describe('Test primary descriptor bubble', () => {
+  resetDB();
+  it('go to a workflow with multiple files', () => {
+    cy.visit('/workflows/github.com/A/l');
+    goToTab('Files');
+    cy.get('[data-cy=primary-descriptor-bubble]').should('be.visible');
+    cy.get('mat-form-field').click();
+    cy.contains('.mat-option-text', 'arguments.cwl').click();
+    cy.get('[data-cy=go-to-primary-icon]').should('be.visible').click();
+    cy.get('[data-cy=primary-descriptor-bubble]').should('be.visible');
```

Comment:
Optionally verify that the button does not exist, `should('not.exist')`, when it's not a primary descriptor.